src/ttcontrol/ttcontrol.py

```diff
@@ -2,16 +2,31 @@
 # Copyright (C) 2024, Tiny Tapeout LTD
 
 import sys
+import machine
+import math
 
 print(f"\nversion={sys.version.split(';')[1].strip()}")
 
 def select_design(design):
    print(f"TODO: SELECT A DESIGN")
    print(f"design={design}")
 
-def set_clock_hz(hz):
-    print(f"TODO: SET CLOCK HZ")
-    print(f"hz={hz}")
+def set_clock_hz(hz, max_rp2040_freq=133_000_000):
+    # Only support integer frequencies
+    freq = int(hz)
+
+    print(f"Setting clock frequency to {freq} Hz")
```

Comment:
IMHO it's better the print messages that are easy to parse later in the JS code. I'm not sure we actually need a message here - errors / warnings are more interesting (then we can display them nicely in the UI), and maybe the rp2040 clock frequency that we have found (so we can store it to config.ini for consumption by the pyboard firmware)

Reply:
OK, that makes sense - I've changed the messages to be name=value